app-misc/openrgb-plugin-visualmap/openrgb-plugin-visualmap-0.9-r1.ebuild

```diff
@@ -17,6 +17,7 @@ KEYWORDS="amd64"
 
 RDEPEND="
 	>=app-misc/openrgb-0.9:=[-qt6(-)]
```

Comment:
It looks like this USEdep could also be cleaned up, now that the transition will not involve an IUSE qt6 (introduced in e2d16fbdfc3b05d8f89591481725b6cc5d6dedd2 for transition) :)